src/authentication/create-account-details/index.test.tsx

```diff
@@ -63,4 +63,18 @@ describe('CreateAccountDetails', () => {
 
     expect(wrapper.find('Alert').prop('children')).toEqual('invalid');
   });
+
+  it('disables button when name is empty', function () {
```

Comment:
Are we sure we want this? In previous discussions with Ethan we decided to always keep the button enabled and the errors would tell the user what's required. The problem now is that there's nothing telling the user which fields are required and why the button is disabled.